steam/webauth.py

```diff
@@ -323,6 +322,57 @@ def _finalize_login(self, login_response):
         self.steam_id = SteamID(data['steamid'])
         self.oauth_token = data['oauth_token']
 
+    def oauth_login(self, oauth_token='', steam_id='', language='english'):
```

Comment:
Could you add a docstring for the method. I suppose mention how/where oauth token is obtained.

Reply:
I've pushed a commit with the docstring, however it doesn't seem to be showing on the PR. Changes are [here](https://github.com/melvyn2/steam/blob/master/steam/webauth.py#L326-L339).